scripts/release/release.sh

```diff
@@ -29,7 +29,7 @@ git push origin ${branch}
 git checkout -b maint_spark2_2.11-${release} ${branch}
 git commit -a -m "Modifying pom.xml files for Spark 2, Scala 2.11 release."
 mvn --batch-mode \
-  -P distribution \
+  -P python,r,distribution \
```

Comment:
On further consideration, this won't work, will it?  The only way I get the python and r profiles to build for me locally is to run separate mvn steps after first running mvn install, and then specify all the `PY4J_ZIP PYSPARK_SUBMIT_ARGS` etc. environment variables.